nz.scuttlebutt.Patchwork.json

```diff
@@ -0,0 +1,177 @@
+{
+    "app-id": "nz.scuttlebutt.Patchwork",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "1.6",
+    "branch": "stable",
+    "sdk": "org.freedesktop.Sdk",
+    "base": "io.atom.electron.BaseApp",
+    "base-version": "stable",
+    "command": "run.sh",
+    "separate-locales": false,
+    "finish-args": [
+        "--device=dri",
+        "--share=ipc",
+        "--share=network",
+        "--socket=pulseaudio",
+        "--socket=wayland",
+        "--socket=x11",
+        "--filesystem=~/.ssb:create",
```

Comment:
Use `--persist` instead (also reporting upstream about XDG_CONFIG_DIR would be good).